apps/system-e2e/src/tests/service-portal/smoke/service-portal.spec.ts

```diff
@@ -21,8 +21,8 @@ test.describe('Service portal', () => {
   test('should have clickable navigation bar', async () => {
     const page = await context.newPage()
     await page.goto('/minarsidur')
-    await expect(
-      await page.locator('a[href^="/minarsidur/"]:has(svg):visible').nth(4),
+    expect(
```

Comment:
This does not feel right, at least one of these should be awaited, otherwise it cannot possible be a retried operation. Can you please double check that?

Reply:
My editor tells me that both `expect` and `page.locator` don't return promises, and therefore awaiting is redundant.

Reply:
`page.locator` returns a non-promise `Locator` ([docs](https://playwright.dev/docs/api/class-page#page-locator))

Since `page.locator` is synchronous `expect` is also synchronous.